### DISCUSSION TITLE
Failed to run `flutter create .` in C:\Users\......\serverpod_learn_flutter

### DISCUSSION THREAD

**Discussion by RisanPT** [serverpod/serverpod#2172] [Q&A] [answered]
**Describe the bug**
When I create a new project using serverpod create,
The flutter create step is failing. 
Similar to issue : #2020 

**terminal output**

Creating Serverpod project "serverpod_learn".

✓ Creating project directories. (3ms)
✓ Writing project files. (0.2s)
✓ Getting server package dependencies. (2.0s)
✓ Getting client package dependencies. (1.0s)
⠦ Getting Flutter app package dependencies.... (2.8s)
**Failed to run `flutter create .` in C:\Users\risan\serverpodprojects\serverpod_learn\serverpod_learn_flutter
✗ Getting Flutter app package dependencies. (2.9s)**
✓ Creating default database migration. (0.3s)

**Serverpod versions**
 -  _CLI Version: 1.2.6
 - _Library version: 1.2.6

**Platform information**
I am using windows 11.

**Comment by RisanPT** [comment]
**Verbose of serverpod create**

Dart SDK version: 3.3.4 (stable) (Tue Apr 16 19:56:12 2024 +0000) on "windows_x64"
Flutter 3.19.6 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 54e66469a9 (5 days ago) • 2024-04-17 13:08:03 -0700
Engine • revision c4cd48e186
Tools • Dart 3.3.4 • DevTools 2.31.1
Creating Serverpod project "test".

⠙ Creating project directories.... (1ms)
 • Creating directory: C:\Users\risan\serverpodprojects\test
 • Creating directory: C:\Users\risan\serverpodprojects\test\test_server
 • Creating directory: C:\Users\risan\serverpodprojects\test\test_client
 • Creating directory: C:\Users\risan\serverpodprojects\test\test_flutter
 • Creating directory: C:\Users\risan\serverpodprojects\test\.github
✓ Creating project directories. (9ms)
⠙ Writing project files.... (0ms)
Copying server files
 • C:\Users\risan\serverpodprojects\test\test_server\analysis_options.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\bin\main.dart
 • C:\Users\risan\serverpodprojects\test\test_server\CHANGELOG.md
 • C:\Users\risan\serverpodprojects\test\test_server\config\development.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\config\generator.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\config\passwords.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\config\production.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\config\staging.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\scripts\appspec.yml
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\scripts\install_dependencies
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\scripts\run_serverpod
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\scripts\start_server
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\balancers-staging.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\balancers.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\cloudfront-web-staging.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\cloudfront-web.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\code-deploy.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\config.auto.tfvars
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\database.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\gitignore
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\init-script.sh
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\instances.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\main.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\redis.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\staging.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\storage.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\variables.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\aws\terraform\vpc.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\gcp\console_gcr\cloud-run-deploy.sh
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\gcp\terraform_gce\config.auto.tfvars
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\gcp\terraform_gce\gitignore
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\gcp\terraform_gce\main.tf
 • C:\Users\risan\serverpodprojects\test\test_server\deploy\gcp\terraform_gce\variables.tf
 • C:\Users\risan\serverpodprojects\test\test_server\docker-compose.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\Dockerfile
 • C:\Users\risan\serverpodprojects\test\test_server\gcloudignore
 • C:\Users\risan\serverpodprojects\test\test_server\generated\tables.pgsql
 • C:\Users\risan\serverpodprojects\test\test_server\gitignore
 • C:\Users\risan\serverpodprojects\test\test_server\lib\server.dart
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\endpoints\example_endpoint.dart
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\future_calls\example_future_call.dart
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\generated\endpoints.dart
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\generated\example.dart
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\generated\protocol.dart
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\generated\protocol.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\models\example.spy.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\web\routes\root.dart
 • C:\Users\risan\serverpodprojects\test\test_server\lib\src\web\widgets\default_page_widget.dart
 • C:\Users\risan\serverpodprojects\test\test_server\pubspec.yaml
 • C:\Users\risan\serverpodprojects\test\test_server\README.md
 • C:\Users\risan\serverpodprojects\test\test_server\web\static\css\style.css
 • C:\Users\risan\serverpodprojects\test\test_server\web\static\images\background.svg
 • C:\Users\risan\serverpodprojects\test\test_server\web\static\images\serverpod-logo.svg
 • C:\Users\risan\serverpodprojects\test\test_server\web\templates\default.html

Copying client files
 • C:\Users\risan\serverpodprojects\test\test_client\analysis_options.yaml
 • C:\Users\risan\serverpodprojects\test\test_client\CHANGELOG.md
 • C:\Users\risan\serverpodprojects\test\test_client\dartdoc_options.yaml
 • C:\Users\risan\serverpodprojects\test\test_client\doc\endpoint.md
 • C:\Users\risan\serverpodprojects\test\test_client\gitignore
 • C:\Users\risan\serverpodprojects\test\test_client\lib\projectname_client.dart
 • C:\Users\risan\serverpodprojects\test\test_client\lib\src\protocol\client.dart
 • C:\Users\risan\serverpodprojects\test\test_client\lib\src\protocol\example.dart
 • C:\Users\risan\serverpodprojects\test\test_client\lib\src\protocol\protocol.dart
 • C:\Users\risan\serverpodprojects\test\test_client\pubspec.yaml
 • C:\Users\risan\serverpodprojects\test\test_client\README.md

Copying Flutter files
 • C:\Users\risan\serverpodprojects\test\test_flutter\analysis_options.yaml
 • C:\Users\risan\serverpodprojects\test\test_flutter\gitignore
 • C:\Users\risan\serverpodprojects\test\test_flutter\lib\main.dart
 • C:\Users\risan\serverpodprojects\test\test_flutter\pubspec.yaml
 • C:\Users\risan\serverpodprojects\test\test_flutter\README.md
 • C:\Users\risan\serverpodprojects\test\test_flutter\test\widget_test.dart

Copying .github files
 • C:\Users\risan\serverpodprojects\test\.github\workflows\deployment-aws.yml
 • C:\Users\risan\serverpodprojects\test\.github\workflows\deployment-gcp.yml
✓ Writing project files. (1.0s)
⠙ Getting server package dependencies.... (0ms)

Running `dart pub get` in C:\Users\risan\serverpodprojects\test\test_server
Resolving dependencies...
+ args 2.5.0
+ async 2.11.0
+ buffer 1.2.3
+ charcode 1.3.1
+ collection 1.18.0
+ crypto 3.0.3
+ executor 2.2.3
+ fixnum 1.1.0
+ http 1.2.1
+ http_parser 4.0.2
+ lints 3.0.0
+ meta 1.14.0
+ mustache_template 2.0.0
+ path 1.9.0
+ pedantic 1.11.1
+ pool 1.5.1
+ postgres 2.6.4 (3.2.0 available)
+ postgres_pool 2.1.6+1
+ redis 4.0.0
+ retry 3.1.2
+ sasl_scram 0.1.1
+ saslprep 1.0.2
+ serverpod 1.2.6
+ serverpod_client 1.2.6
+ serverpod_serialization 1.2.6
+ serverpod_service_client 1.2.6
+ serverpod_shared 1.2.6
+ source_span 1.10.0
+ sprintf 7.0.0
+ stack_trace 1.11.1
+ stream_channel 2.1.2
+ string_scanner 1.2.0
+ super_string 1.0.3
+ synchronized 3.1.0+1
+ system_resources 1.6.0
+ term_glyph 1.2.1
+ typed_data 1.3.2
+ unorm_dart 0.2.0 (0.3.0 available)
+ uuid 4.4.0
+ vm_service 14.2.1
+ web 0.5.1
+ web_socket_channel 2.4.5
+ yaml 3.1.2
Changed 43 dependencies!
2 packages have newer versions incompatible with dependency constraints.
Try `dart pub outdated` for more information.
✓ Getting server package dependencies. (2.4s)
⠙ Getting client package dependencies.... (0ms)

Running `dart pub get` in C:\Users\risan\serverpodprojects\test\test_client
Resolving dependencies...
+ async 2.11.0
+ collection 1.18.0
+ crypto 3.0.3
+ fixnum 1.1.0
+ http 1.2.1
+ http_parser 4.0.2
+ meta 1.14.0
+ path 1.9.0
+ serverpod_client 1.2.6
+ serverpod_serialization 1.2.6
+ source_span 1.10.0
+ sprintf 7.0.0
+ stream_channel 2.1.2
+ string_scanner 1.2.0
+ term_glyph 1.2.1
+ typed_data 1.3.2
+ uuid 4.4.0
+ web 0.5.1
+ web_socket_channel 2.4.5
+ yaml 3.1.2
Changed 20 dependencies!
✓ Getting client package dependencies. (1.1s)
⠙ Getting Flutter app package dependencies.... (0ms)

Running `flutter create .` in C:\Users\risan\serverpodprojects\test\test_flutter
Recreating project ....
  .idea\libraries\Dart_SDK.xml (created)
  .idea\libraries\KotlinJavaRuntime.xml (created)
  .idea\modules.xml (created)
  .idea\runConfigurations\main_dart.xml (created)
  .idea\workspace.xml (created)
  android\app\build.gradle (created)
  android\app\src\main\kotlin\com\example\test_flutter\MainActivity.kt (created)
  android\build.gradle (created)
  android\test_flutter_android.iml (created)
  android\.gitignore (created)
  android\app\src\debug\AndroidManifest.xml (created)
  android\app\src\main\AndroidManifest.xml (created)
  android\app\src\main\res\drawable\launch_background.xml (created)
  android\app\src\main\res\drawable-v21\launch_background.xml (created)
  android\app\src\main\res\mipmap-hdpi\ic_launcher.png (created)
  android\app\src\main\res\mipmap-mdpi\ic_launcher.png (created)
  android\app\src\main\res\mipmap-xhdpi\ic_launcher.png (created)
  android\app\src\main\res\mipmap-xxhdpi\ic_launcher.png (created)
  android\app\src\main\res\mipmap-xxxhdpi\ic_launcher.png (created)
  android\app\src\main\res\values\styles.xml (created)
  android\app\src\main\res\values-night\styles.xml (created)
  android\app\src\profile\AndroidManifest.xml (created)
  android\gradle\wrapper\gradle-wrapper.properties (created)
  android\gradle.properties (created)
  android\settings.gradle (created)
  ios\Runner\AppDelegate.swift (created)
  ios\Runner\Runner-Bridging-Header.h (created)
  ios\Runner.xcodeproj\project.pbxproj (created)
  ios\Runner.xcodeproj\xcshareddata\xcschemes\Runner.xcscheme (created)
  ios\RunnerTests\RunnerTests.swift (created)
  ios\.gitignore (created)
  ios\Flutter\AppFrameworkInfo.plist (created)
  ios\Flutter\Debug.xcconfig (created)
  ios\Flutter\Release.xcconfig (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Contents.json (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-1024x1024@1x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-20x20@1x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-20x20@2x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-20x20@3x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-29x29@1x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-29x29@2x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-29x29@3x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-40x40@1x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-40x40@2x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-40x40@3x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-60x60@2x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-60x60@3x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-76x76@1x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-76x76@2x.png (created)
  ios\Runner\Assets.xcassets\AppIcon.appiconset\Icon-App-83.5x83.5@2x.png (created)
  ios\Runner\Assets.xcassets\LaunchImage.imageset\Contents.json (created)
  ios\Runner\Assets.xcassets\LaunchImage.imageset\LaunchImage.png (created)
  ios\Runner\Assets.xcassets\LaunchImage.imageset\LaunchImage@2x.png (created)
  ios\Runner\Assets.xcassets\LaunchImage.imageset\LaunchImage@3x.png (created)
  ios\Runner\Assets.xcassets\LaunchImage.imageset\README.md (created)
  ios\Runner\Base.lproj\LaunchScreen.storyboard (created)
  ios\Runner\Base.lproj\Main.storyboard (created)
  ios\Runner\Info.plist (created)
  ios\Runner.xcodeproj\project.xcworkspace\contents.xcworkspacedata (created)
  ios\Runner.xcodeproj\project.xcworkspace\xcshareddata\IDEWorkspaceChecks.plist (created)
  ios\Runner.xcodeproj\project.xcworkspace\xcshareddata\WorkspaceSettings.xcsettings (created)
  ios\Runner.xcworkspace\contents.xcworkspacedata (created)
  ios\Runner.xcworkspace\xcshareddata\IDEWorkspaceChecks.plist (created)
  ios\Runner.xcworkspace\xcshareddata\WorkspaceSettings.xcsettings (created)
  linux\.gitignore (created)
  linux\CMakeLists.txt (created)
  linux\flutter\CMakeLists.txt (created)
  linux\main.cc (created)
  linux\my_application.cc (created)
  linux\my_application.h (created)
  macos\.gitignore (created)
  macos\Flutter\Flutter-Debug.xcconfig (created)
  macos\Flutter\Flutter-Release.xcconfig (created)
  macos\Runner\AppDelegate.swift (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\app_icon_1024.png (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\app_icon_128.png (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\app_icon_16.png (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\app_icon_256.png (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\app_icon_32.png (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\app_icon_512.png (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\app_icon_64.png (created)
  macos\Runner\Assets.xcassets\AppIcon.appiconset\Contents.json (created)
  macos\Runner\Base.lproj\MainMenu.xib (created)
  macos\Runner\Configs\AppInfo.xcconfig (created)
  macos\Runner\Configs\Debug.xcconfig (created)
  macos\Runner\Configs\Release.xcconfig (created)
  macos\Runner\Configs\Warnings.xcconfig (created)
  macos\Runner\DebugProfile.entitlements (created)
  macos\Runner\Info.plist (created)
  macos\Runner\MainFlutterWindow.swift (created)
  macos\Runner\Release.entitlements (created)
  macos\Runner.xcodeproj\project.pbxproj (created)
  macos\Runner.xcodeproj\project.xcworkspace\xcshareddata\IDEWorkspaceChecks.plist (created)
  macos\Runner.xcodeproj\xcshareddata\xcschemes\Runner.xcscheme (created)
  macos\Runner.xcworkspace\contents.xcworkspacedata (created)
  macos\Runner.xcworkspace\xcshareddata\IDEWorkspaceChecks.plist (created)
  macos\RunnerTests\RunnerTests.swift (created)
  test_flutter.iml (created)
  web\favicon.png (created)
  web\icons\Icon-192.png (created)
  web\icons\Icon-512.png (created)
  web\icons\Icon-maskable-192.png (created)
  web\icons\Icon-maskable-512.png (created)
  web\index.html (created)
  web\manifest.json (created)
  windows\.gitignore (created)
  windows\CMakeLists.txt (created)
  windows\flutter\CMakeLists.txt (created)
  windows\runner\CMakeLists.txt (created)
  windows\runner\flutter_window.cpp (created)
  windows\runner\flutter_window.h (created)
  windows\runner\main.cpp (created)
  windows\runner\resource.h (created)
  windows\runner\resources\app_icon.ico (created)
  windows\runner\runner.exe.manifest (created)
  windows\runner\Runner.rc (created)
  windows\runner\utils.cpp (created)
  windows\runner\utils.h (created)
  windows\runner\win32_window.cpp (created)
  windows\runner\win32_window.h (created)
Resolving dependencies...
Got dependencies.
Building with plugins requires symlink support.

Please enable Developer Mode in your system settings. Run
  start ms-settings:developers
to open settings.
Failed to run `flutter create .` in C:\Users\risan\serverpodprojects\test\test_flutter
✗ Getting Flutter app package dependencies. (3.8s)
⠙ Creating default database migration.... (0ms)
Creating initial migration.
✓ Creating default database migration. (0.3s)

**Comment by RisanPT** [comment]
**After the project is created I did flutter create . inside the flutter project folder, no failures. It just works fine**

**Comment by SandPod** [comment, marked as answer]
Hey @RisanPT!

Glad to see you are learning Serverpod 🚀 

It seems like the issue is printed in the verbose output:

> Please enable Developer Mode in your system settings. Run
start ms-settings:developers
to open settings.

You might need to enable apps to be installed from anywhere in developer mode in windows as discussed here: https://stackoverflow.com/questions/68089177/flutter-building-with-plugins-requires-symlink-support
